### ISSUE TITLE
Spotify won't start, errors with: Failed to call service spotcast/start. "name='wp_access_token', domain=None, path=None"

### ISSUE THREAD

**Issue by mondomx** [custom-cards/spotify-card#29]
Hello,

I'm having issues connecting to any chromecast devices (that are not previously connected), they appear listed but when I click on them it fails to callthe service. For example if I click on "Home Group" for the google chromecasts group it just errors out and this comes up in the homeassistant.log

```
2019-10-01 17:53:45 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection.1871198480] "name='wp_access_token', domain=None, path=None"
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/websocket_api/commands.py", line 130, in handle_call_service
    connection.context(msg),
  File "/usr/src/homeassistant/homeassistant/core.py", line 1234, in async_call
    await asyncio.shield(self._execute_service(handler, service_call))
  File "/usr/src/homeassistant/homeassistant/core.py", line 1261, in _execute_service
    await self._hass.async_add_executor_job(handler.func, service_call)
  File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/spotcast/__init__.py", line 162, in start_casting
    access_token, expires = get_spotify_token(username=user, password=pwd)
  File "/config/custom_components/spotcast/__init__.py", line 85, in get_spotify_token
    data = st.start_session(username, password)
  File "/usr/local/lib/python3.7/site-packages/spotify_token.py", line 70, in start_session
    access_token, expiration_date = _get_access_token(session, cookies)
  File "/usr/local/lib/python3.7/site-packages/spotify_token.py", line 42, in _get_access_token
    access_token = response.cookies['wp_access_token']
  File "/usr/local/lib/python3.7/site-packages/requests/cookies.py", line 328, in __getitem__
    return self._find_no_duplicates(name)
  File "/usr/local/lib/python3.7/site-packages/requests/cookies.py", line 399, in _find_no_duplicates
    raise KeyError('name=%r, domain=%r, path=%r' % (name, domain, path))
KeyError: "name='wp_access_token', domain=None, path=None"
```

**Comment by niccolobusato** [comment]
i have the same problem!

**Comment by fondberg** [comment]
Please note that there is an ongoing issue with spotcast. Head over to that repo to monitor it.
Until that is fixed chromecast will not work from this card

**Comment by fondberg** [comment]
This is fixed in spotcast version 2.3
